supporting_content/appendix/quantity_driven_pricing_strategies.md

```diff
@@ -45,26 +45,27 @@ For better comprehension, please refer to the sample price-tiers configuration a
-| :-------- | :--------------------------------------------------------------------------------- | :-------------------------------------------------------------------- | :---------------------- |
-| AWS       | Flat-rate based pricing<br>SKU: E9YHNFENF4XQBZR6                                   | pricing/publicOnDemandRate: 0.000005                                  | _TODO: Add sample data_ |
-| AWS       | Quantity-dependent pricing<br> ??? _TODO: look for a higher tier sample_           | pricing/publicOnDemandRate: ??? _TODO: look for a higher tier sample_ | _TODO: Add sample data_ |
-| GCP       | Flat-rate based pricing                                                            | Not available                                                         | _TODO: Add sample data_ |
-| GCP       | Quantity-dependent pricing                                                         | Not available                                                         | _TODO: Add sample data_ |
-| Microsoft | Flat-rate based pricing - PAYG<br>meterId: b9e5e77c-a0b3-4a2c-9b8b-57fa54f31c52    | PayGPrice: 0.00036                                                    | _TODO: Add sample data_ |
-| Microsoft | Flat-rate based pricing - CSP<br>meterId: b9e5e77c-a0b3-4a2c-9b8b-57fa54f31c52     | PayGPrice: 0.0003707                                                  | _TODO: Add sample data_ |
-| Microsoft | Quantity-dependent pricing - PAYG<br>meterId: 9995d93a-7d35-4d3f-9c69-7a7fea447ef4 | PayGPrice: 0.087                                                      | _TODO: Add sample data_ |
-| Microsoft | Quantity-dependent pricing - CSP<br>meterId: 9995d93a-7d35-4d3f-9c69-7a7fea447ef4  | PayGPrice: 0.087                                                      | _TODO: Add sample data_ |
-| OCI       | Flat-rate based pricing                                                            | Not available                                                         | _TODO: Add sample data_ |
-| OCI       | Quantity-dependent pricing                                                         | Not available                                                         | _TODO: Add sample data_ |
+| Provider  | Scenario                                                                           | ListUnitPrice Pattern                                                 |
+| :-------- |:---------------------------------------------------------------------------------- | :-------------------------------------------------------------------- |
+| AWS       | Flat-rate based pricing<br>SKU: E9YHNFENF4XQBZR6                                   | pricing/publicOnDemandRate: 0.000005                                  |
+| AWS       | Quantity-dependent pricing<br> ??? _TODO: look for a higher tier sample_           | pricing/publicOnDemandRate: ??? _TODO: look for a higher tier sample_ |
```

Comment:
Would be fantastic to go to market in 1.0 without this "TODO" here.

Reply:
As this pending TODO is part of supporting content, we are proceeding with the merge. However, it is acknowledged that there is a requirement to acquire and provide a higher-tier sample before advancing to market with the specification.